polaris/polaris/tests/sep24/test_deposit.py

```diff
@@ -23,6 +23,7 @@
     "successful": True,
     "id": "test_stellar_id",
     "paging_token": "123456789",
+    "result_xdr": "AAAAAAAAAGQAAAAAAAAAAQAAAAAAAAAOAAAAAAAAAAAyBzvi/vP0Bih6bAqRNkiutMVUkW1S+WtuITJAA2LOjgAAAAA=",
```

Comment:
I'd prefer to have a difference success response object used for CBs, since most success responses will have `result_xdr`s for payment ops.

Reply:
I added the `results_xdr` key value in response to a comment you made in the last CB attempt
https://github.com/stellar/django-polaris/pull/341#discussion_r509778548

Would you prefer to keep it how it is currently or have a a difference success response object?

Reply:
I decided to go ahead and make a `HORIZON_SUCCESS_RESPONSE_CLAIM` object to avoid `result_xdr` confusion.

Reply:
Lol welp, that was dumb of me. I'll leave this to you.

Reply:
Lol, nah its chill I'll keep them separate.
It's cleaner this way imo